Sources/Clickstream/Core/Interface/ClickStream.swift

```diff
@@ -8,9 +8,42 @@
 import Foundation
 import SwiftProtobuf
 
+/// Conform to this delegate to send the current user location details and set NTP Time.
+public protocol ClickStreamDataSource: AnyObject {
```

Comment:
Lowercase `S` in ClickStreamDataSource.

We are following `Clickstream` format all across